frontend/src/assets/styles/colours.css

```diff
@@ -80,6 +80,16 @@ html {
   --colour-ti-critical: #DC2626;
   --colour-ti-success: #1D783B;
 
+  /*
+   * Form elements
+   * Applied frequently for inputs, switches and interactive elements.
+   */
+  --colour-form-base: #E4E4E7;
```

Comment:
I believe this is actually element specific. I would maybe move these to the SwitchToggle. 

But I'll cc @laurelterlesky for comment.